src/components/Button/Button.tsx

```diff
@@ -170,6 +170,7 @@ export function Button({
         className={classNames(
           styles.DisclosureIcon,
           disclosure === 'up' && styles.DisclosureIconFacingUp,
+          loading && styles.Hidden
```

Comment:
Missed this one in the previous PR, the disclosure icon was also flashing a black square.